cmd/main.go

```diff
@@ -298,12 +264,6 @@ func main() {
 	hostStateMonitor.Start()
 	defer hostStateMonitor.Stop()
 
-	if newUrl, err = s3wrapper.FixEndpointURL(Options.BMConfig.S3EndpointURL); err != nil {
```

Comment:
Was it removed on purpose?

Reply:
I see the same call here https://github.com/openshift/assisted-service/pull/830/files#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R191 
It wasn't clear to me why it's called twice.

Reply:
That's `Options.JobConfig.S3EndpointURL` and `Options.BMConfig.S3EndpointURL` .